### PR TITLE
Fix page form change detection

### PR DESCRIPTION
**Pull request by melegiul**
### Short description

Fix page form change detection

### Proposed changes

Coerce initial and data value to integers
hasChanged

### Resolved issues

Fixes: #1251

### CODE REVIEW

**Comment by ulliholtgrave** [comment]
Very good catch @melegiul ! :+1: Can you add a notice to the Changelog that the issue is solved? :)

**Comment by melegiul** [comment]
Thank you for your review, I have updated the changelog

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ UNRELEASED
 * [ [#1131](https://github.com/digitalfabrik/integreat-cms/issues/1131) ] Flush Cache of related objects when changing a tree
 * [ [#1242](https://github.com/digitalfabrik/integreat-cms/issues/1242) ] Add setting to activate Matomo tracking
 * [ [#1197](https://github.com/digitalfabrik/integreat-cms/issues/1197) ] Fix calculation of translation status
+* [ [#1251](https://github.com/digitalfabrik/integreat-cms/issues/1251) ] Fix change detection in page form
```
Comment (timoludwig) [inline CHANGELOG.md:13]: Could you add the changelog to the "unreleased" section please? (this probably requires a rebase on develop to resolve the conflict with the recently merged PR)

**Review by timoludwig — APPROVED** [review]
:+1:

**Pull request merged**